setup.cfg

```diff
@@ -37,9 +37,9 @@ license = AGPLv3
 url = TBD
 edit_on_github = False
 github_project = opencadc/dao2caom2
-install_requires = caom2utils caom2repo cadcdata
+install_requires = cadcdata cadctap caom2 caom2repo caom2utils importlib-metadata python-dateutil PyYAML spherical-geometry vos
 # version should be PEP386 compatible (http://www.python.org/dev/peps/pep-0386)
-version = 0.6
+version = 0.7
```

Comment:
These changes don't look backward compatible, should this maybe be 1.0 now, or are you the only one who cares about the version?

Reply:
The versions show up in CAOM2 records, but the only thing that matters about the versions is that they're different.